src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -188,6 +198,32 @@ the main body of the message as well as a quote.
 				<span>{{ t('spreed', 'Unread messages') }}</span>
 			</div>
 		</div>
+
+		<!-- Message forwarding -->
+		<RoomSelector
+			v-if="showRoomSelector"
+			container="#content-vue"
+			:show-postable-only="true"
+			:dialog-title="dialogTitle"
+			:dialog-subtitle="dialogSubtitle"
+			@select="setSelectedConversation"
+			@close="showRoomSelector=false" />
+		<Modal v-if="showForwardedConfirmation"
+			@close="showForwardedConfirmation = false">
+			<EmptyContent icon="icon-checkmark" class="forwarded-confirmation__emptycontent">
+				<template #desc>
+					{{ t('spreed', 'The message has been forwarded to {selectedConversation}', { selectedConversation }) }}
+				</template>
+			</EmptyContent>
+			<div class="forwarded-confirmation__navigation">
+				<button @click="showForwardedConfirmation=false">
+					{{ t('spreed', 'Dismiss') }}
+				</button>
+				<button class="primary" @click="openConversation(room)">
+					{{ t('spreed', 'Go to conversation') }}
+				</button>
+			</div>
+		</Modal>
```

Comment:
I would accept this if you ask for confirmation. But otherwise I guess showSuccess should be enough?